package_manifest.yaml

```diff
@@ -327,19 +362,69 @@ katello_packages:
     rubygem-redhat_access_lib: {}
     rubygem-robotex: {}
     rubygem-runcible: {}
+
 katello_client_packages:
   vars:
     package_base_dir: 'packages/katello/'
     releasers:
     - koji-katello-client
+    repoclosure_config: repoclosure/yum.conf
```

Comment:
Doesn't feel right to group all distros together. Perhaps this is really a limitation of the modeling we chose (and why it's always good to have more than 1 distro - your models are better).

Reply:
It doesn't feel right but yum/repoclosure is smarter than I even thought. If we don't lump them together, then we have to identify and download each task per distro which is tedious. However, repoclosure if you point it at a directory takes distro into account for us.